sdk/formrecognizer/test-resources.json

```diff
@@ -9,7 +9,7 @@
                 "description": "The base resource name."
             }
         },
-        "location": {
+        "formRecognizerLocation": {
             "type": "string",
             "defaultValue": "canadacentral",
```

Comment:
by having here a default value, will the `Clouds` parameter change the region accordingly? or will these region be always used and we should remove it from here

Reply:
I was wondering this too. @tzhanl @Luyunmt ?

Reply:
Now we don't need change region, removing this change.